### PR TITLE
Use eslint-config-metarhia and fix linting issues

### PR DESCRIPTION
**Pull request by belochub**
Refs: https://github.com/metarhia/Metarhia/issues/22

### CODE REVIEW

File: lib/array.js [hunk 1]
```diff
@@ -11,11 +11,10 @@ const splitAt = (
...
 
-const shuffle = (
-  // Shuffle an array
-  arr // array
-  // Returns: array
-) => (
+// Shuffle an array
+//   arr - array
+// Returns: array
+const shuffle = arr => (
```
Comment (tshemsedinov) [inline lib/array.js]: Better remove `()` in such cases
```suggestion
const shuffle = arr => arr.sort(() => Math.random() - 0.5);
```

File: lib/callbacks.js [hunk 2]
```diff
@@ -18,10 +18,9 @@ const emptiness = (
   // Returns: always undefined
 ) => {};
 
-const nop = (
-  // Empty asynchronous callback-last single-argument function
-  callback // function, callback to be called with (null)
-) => {
+// Empty asynchronous callback-last single-argument function
+//   callback - function, callback to be called with (null)
+const nop = callback => {
```
Comment (tshemsedinov) [inline lib/callbacks.js]: This case can't be optimized to single-line

File: lib/array.js [hunk 3]
```diff
@@ -59,11 +58,10 @@ const sequence = (
...
 
-const last = (
-  // Get last element of array
-  arr // array
-  // Returns: element
-) => (
+// Get last element of array
+//   arr - array
+// Returns: element
+const last = arr => (
```
[the remark this answers is not in the document]
Comment (tshemsedinov) [reply lib/array.js]: Yes, I mean in all case like this, Nick knows

File: lib/auth.js [hunk 4]
```diff
@@ -65,16 +65,14 @@ const passwordTests = {
     options: { number: 2 },
   },
   MIN_LOWERCASE_CHARS: {
-    test: (password, option) => (
-      stringIncludesChars(password, unicodeCategories.Ll, option.number)
-    ),
+    test: (password, option) =>
```
Comment (tshemsedinov) [inline lib/auth.js]: I'd prefer:
```js
test: (password, option) => stringIncludesChars(
  password, unicodeCategories.Ll, option.number
),
```

File: lib/auth.js [hunk 5]
```diff
@@ -65,16 +65,14 @@ const passwordTests = {
...
+      stringIncludesChars(password, unicodeCategories.Ll, option.number),
     hint: options => ({ name: 'MIN_LOWERCASE_CHARS', number: options.number }),
     options: { number: 1 },
   },
   MIN_UPPERCASE_CHARS: {
-    test: (password, options) => (
-      stringIncludesChars(password, unicodeCategories.Lu, options.number)
-    ),
+    test: (password, options) =>
```
Comment (tshemsedinov) [inline lib/auth.js]: ditto

File: lib/strings.js [hunk 6]
```diff
@@ -21,9 +21,8 @@ const ALPHA_DIGIT = ALPHA + DIGIT;
 //   content - string, to escape
 // Returns: string
 // Example: htmlEscape('5>=5') = '5&lt;=5'
-const htmlEscape = content => (
-  content.replace(HTML_ESCAPE_REGEXP, char => HTML_ESCAPE_CHARS[char])
-);
+const htmlEscape = content =>
+  content.replace(HTML_ESCAPE_REGEXP, char => HTML_ESCAPE_CHARS[char]);
```
Comment (tshemsedinov) [inline lib/strings.js]:
```js
const htmlEscape = content => content.replace(
  HTML_ESCAPE_REGEXP,
  char => HTML_ESCAPE_CHARS[char]
);
```

File: lib/strings.js [hunk 7]
```diff
@@ -92,9 +91,8 @@ const subst = (
 // Returns: string
 // Example: fileExt('/dir/file.txt')
 // Result: 'txt'
-const fileExt = fileName => (
-  path.extname(fileName).replace('.', '').toLowerCase()
-);
+const fileExt = fileName =>
+  path.extname(fileName).replace('.', '').toLowerCase();
```
Comment (tshemsedinov) [inline lib/strings.js]:
```js
const fileExt = fileName => path.extname(fileName)
  .replace('.', '').toLowerCase();
```

File: lib/strings.js [hunk 8]
```diff
@@ -108,24 +106,20 @@ const CAPITALIZE_REGEXP = /\w+/g;
 // Capitalize string
 //   s - string
 // Returns: string
-const capitalize = s => (
-  s.replace(CAPITALIZE_REGEXP, word => (
+const capitalize = s =>
+  s.replace(CAPITALIZE_REGEXP, word =>
     word.charAt(0).toUpperCase() + word.substr(1).toLowerCase()
```
Comment (tshemsedinov) [inline lib/strings.js]:
```js
const capitalize = s => s.replace(
  CAPITALIZE_REGEXP,
  word => word.charAt(0).toUpperCase() + word.substr(1).toLowerCase()
);
```

**Comment by tshemsedinov** [comment]
@belochub please land this

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/common/commit/5ae54f21001f619fc6641f225a67c92edf43ff65.

**Pull request closed without merging**